fedora_bot.py

```diff
@@ -292,8 +292,9 @@ def main():
     for component_numtests in args.component:
         try:
             component, num_tests = component_numtests.split(':')
-        except AttributeError:
-            parser.error(f"Invalid component format, must be PACKAGE:NUM_TESTS: {component_numtests}")
+            num_tests = int(num_tests)
+        except ValueError:
+            parser.error(f"Invalid component format, must be PACKAGE:NUM_TESTS : {component_numtests}")
```

Comment:
FTR, I added that space to avoid ambiguity: the first colon is part of the value syntax, the second is just separating the error message from the provided value.